images/build/Dockerfiles/vso.Dockerfile

```diff
@@ -75,6 +75,4 @@ RUN --mount=type=secret,id=oryx_sdk_storage_account_access_token \
     && cd /opt/maven \
     && ln -s $MAVEN_VERSION lts \
     && rm -rf /opt/tmp \
-    && echo "vso" > /opt/oryx/.imagetype \
-    && echo "DEBIAN|${DEBIAN_FLAVOR}" | tr '[a-z]' '[A-Z]' > /opt/oryx/.ostype \
```

Comment:
Same comment here -- line was accidentally removed

Reply:
thanks for finding it. fixed